### ISSUE TITLE
Enable a11y checks in RevisionCardFragmentTest [Blocked on #4235]

### ISSUE THREAD

**Issue by JishnuGoyal** [oppia/oppia-android#4631]
The tests under RevisionCardFragmentTest have accessibility checks ignored using the annotation `@DisableAccessibilityChecks` which should be removed once #4235 is resolved.

As we remove `DisableAccessibilityChecks` and run Espresso tests we might notice some more test failures because of accessibility which will need to be tackled and fixed.

**Comment by TanishMoral11** [comment]
Hey @JishnuGoyal, Any Update Regarding This Issue ?

**Comment by JishnuGoyal** [comment]
Hi @adhiamboperes , would you be able to check this?
